definitions/divorce/json/CaseField/CaseField-general-referral-nonprod.json

```diff
@@ -101,5 +101,21 @@
     "Label": "General application added date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "21/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferralWithoutNoticeFeeSummary",
+    "Label": "Here are your payment details",
+    "FieldType": "OrderSummary",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ReferralAmountToPayGeneralLabel",
+    "Label": "Amount to pay: **£${solApplicationFeeInPounds}**",
```

Comment:
are you sure `solApplicationFeeInPounds` is the one you should use here?

Reply:
updated